src/pages/ProviderEServiceDetailsPage/components/ProviderEServiceDetailsTab/ProviderEServiceGeneralInfoSection/ProviderEServiceGeneralInfoSection.tsx

```diff
@@ -101,14 +103,30 @@ export const ProviderEServiceGeneralInfoSection: React.FC = () => {
     label: t('bottomActions.exportVersion'),
   }
 
+  const watchRiskyAnalysisAssociatedAction = {
+    startIcon: <InsertLinkIcon fontSize="small" />,
+    component: 'button',
+    onClick: () =>
+      navigate('WATCH_RISK_ANALISIS_FOR_ESERVICE', {
+        params: {
+          descriptorId: descriptor.id,
+          eserviceId: descriptor.eservice.id,
+        },
+      }),
+    label: t('bottomActions.watchRiskyAnalysisAssociated'),
+  }
```

Comment:
Navigation as a button with an `onClick` is an anti pattern, can we make this as link? 🙏